src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -238,14 +238,18 @@ private synchronized int execMatlabCommand(FilePath workspace, Launcher launcher
         final String uniqueTmpFldrName = getUniqueNameForRunnerFile();
         ProcStarter matlabLauncher;
         try {
-            matlabLauncher = getProcessToRunMatlabCommand(workspace, launcher, listener, envVars,
-                    constructCommandForTest(getInputArguments()), uniqueTmpFldrName);
+            // Get decorated launcher for Windows
+            launcher = getDecoratedLauncherForWindows(launcher);

```

Comment:
The decoration of cmd for Windows was earlier encapsulated by the code that created the ProcStarter. Now it has leaked into the execMATLABCommand. Why is this needed?

Reply:
Removed decoration usage as it was coming from Launcher now handling it in cmds() method now .